nisystemlink/clients/dataframe/models/_query_tables_request.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from typing import List, Optional, Union
+
+from nisystemlink.clients.core._uplink._with_paging import WithPaging
+from pydantic import StrictBool, StrictInt
+
+from ._order_by import OrderBy
+
+
+class QueryTablesRequest(WithPaging):
+    """Request parameters for querying tables."""
+
+    filter: str
+    """The table query filter in Dynamic LINQ format.
```

Comment:
Is there a way to insert a link in the generated documentation? It may be useful to link to the [Dynamic Linq](https://github.com/ni/systemlink-OpenAPI-documents/wiki/Dynamic-Linq-Query-Language) documentation like we do in the swagger UI. In particular, it gives the format for DateTime and TimeSpan filter values and available operators/methods.

Reply:
Should be linking out to that page now